### PR TITLE
feature(function): add custom functions documentation for AWS Amplify-backend

### PR DESCRIPTION
**Pull request by MarlonJD**
… Gen 2 @aws-amplify/amplify-backend/1602

#### Description of changes:
Add custom runtime documents, it's required for [#1602](https://github.com/aws-amplify/amplify-backend/pull/1602)

#### Related GitHub issue
[#1543](https://github.com/aws-amplify/amplify-backend/issues/1543), [#1486](https://github.com/aws-amplify/amplify-backend/issues/1486)

### Instructions

Which product(s) are affected by this PR (if applicable)?
- [x] amplify-backend
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [ ] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [x] JS
- [x] Swift
- [x] Android
- [x] Flutter
- [x] React Native

**Please add the product(s)/platform(s) affected to the PR title**
amplify-backend

#### Checks

- [X] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

### When this PR is ready to merge, please check the box below
- [X] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+  };
+}
+
+AWS Amplify Gen 2 Functions are AWS Lambda functions that can be used to perform tasks and customize workflows in your Amplify app.Functions can be written in Node.js, Python, Go, or any other language supported by AWS Lambda:
+
+<Callout warning>
+
+**Note:** Custom runtimes are not supported in Amplify Functions directly. If you need docker support to use a custom runtime for example in Python, Docker is not supported in Amplify Hosting and Amplify backend auto build. You shouldn't use docker build in your function.There is an example of how to use python in a lambda function in the without Docker section below.
+
+</Callout>
```
Comment (sobolk) [inline src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx:39]:
```suggestion
<Callout warning>

**Note:** [Fullstack Git-based environments](https://docs.amplify.aws/react/how-amplify-works/concepts/#fullstack-git-based-environments) do not support Docker for functions bundling out of the box.

</Callout>
```

File: src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+```txt title="amplify/functions/say-hello/requirements.txt"
+request==2.25.1
+some-other-package>=1.0.0
+```
+
+You're now ready to deploy your python function. Next is the same process as the Node.js/TypeScript function. Go to [Common steps for all languages](#common-steps-for-all-languages) to continue.
+
+# Go functions
+To get started, install go package from npm. go to your backend directory, you should see amplify folder inside it. and run the following command:
+```npm i @aws-cdk/aws-lambda-go-alpha```
```
Comment (sobolk) [inline src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx:108]: We should refrain from using alpha packages in docs.

We should:
- either convert this sample to use `Code.from..` and plain `Function` (like in case of python).
- or just remove Go sample.

**Review by sobolk — COMMENTED** [review]
I left two comments around docker and go. But I suggest to wait until @josefaidt gets a chance to review this before making changes.

File: src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx (hunk 2 above)
Comment (MarlonJD) [reply src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx:108]: Here is an example of how this can be implemented using `Function`:

```
import { defineFunction } from "@aws-amplify/backend";
import { DockerImage, Duration } from "aws-cdk-lib";
import { Code, Function, Runtime } from "aws-cdk-lib/aws-lambda";
import { execSync } from "child_process";
import * as path from "path";
import { fileURLToPath } from "url";

const functionDir = path.dirname(fileURLToPath(import.meta.url));

export const sendSmsGoFunctionHandler = defineFunction(
  (scope) =>
    new Function(scope, "SendSmsGo", {
      handler: "bootstrap",
      runtime: Runtime.PROVIDED_AL2023, // or any other python version
      timeout: Duration.seconds(3), //  default is 3 seconds
      code: Code.fromAsset(functionDir, {
        bundling: {
          image: DockerImage.fromRegistry("dummy"),
          local: {
            tryBundle(outputDir: string) {
              execSync(`rsync -rLv ${functionDir}/* ${path.join(outputDir)}`);
              execSync(
                `cd ${path.join(outputDir)} && GOARCH=amd64 GOOS=linux go build -tags lambda.norpc -o ${path.join(outputDir)}/bootstrap ${functionDir}/main.go`
              );
              return true;
            },
          },
        },
      }),
    }),
);
```

File: src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+              return true;
+            },
+          },
+        },
+      }),
+    })
+);
+```
+
+Next, create the corresponding handler file at `amplify/functions/say-hello/handler.ts`. This is where your function code will go.
```
Comment (rdsimp) [inline src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx:80]: Shouldn't this be `amplify/functions/say-hello/index.py` instead of `amplify/functions/say-hello/handler.ts`?

Comment (MarlonJD) [reply src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx:80]: You're right, it's typo, I'm fixing it

File: src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+export function getStaticProps(context) {
+  return {
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+
+AWS Amplify Gen 2 Functions are AWS Lambda functions that can be used to perform tasks and customize workflows in your Amplify app.Functions can be written in Node.js, Python, Go, or any other language supported by AWS Lambda:
```
Comment (josefaidt) [inline src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx:33]:
```suggestion
AWS Amplify Gen 2 functions are AWS Lambda functions that can be used to perform tasks and customize workflows in your Amplify app. Functions can be written in Node.js, Python, Go, or any [other language supported by AWS Lambda](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html).
```

File: src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx [hunk 5]
```diff
@@ -0,0 +1,238 @@
...
+
+<Callout warning>
+
+**Note:** Custom runtimes are not supported in Amplify Functions directly. If you need docker support to use a custom runtime for example in Python, Docker is not supported in Amplify Hosting and Amplify backend auto build. You shouldn't use docker build in your function.There is an example of how to use python in a lambda function in the without Docker section below.
+
+</Callout>
+
+Technically, you can use any language supported by AWS Lambda. These are NodeJS, Python, Java, .NET, Ruby.
+To use other languages in Lambda, such as Go or Rust, use an OS-only runtime.
+
```
Comment (josefaidt) [inline src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx:43]:
```suggestion
```

File: src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx [hunk 6]
```diff
@@ -0,0 +1,238 @@
...
+In this guide, you will learn how to create python and golang functions in Amplify Functions.
```
Comment (josefaidt) [inline src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx:44]:
```suggestion
In this guide, you will learn how to create Python and Go functions with Amplify functions.
```

File: src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx [hunk 7]
```diff
@@ -0,0 +1,238 @@
...
+
+# Python functions
+To get started, create a new directory and a resource file, `amplify/functions/say-hello/resource.ts`. Then, define the Function with `defineFunction`:
```
Comment (josefaidt) [inline src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx:47]:
```suggestion
## Python

To get started, create a new directory and a resource file, `amplify/functions/say-hello/resource.ts`. Then, define the function with `defineFunction`:
```

File: src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx [hunk 8]
```diff
@@ -0,0 +1,238 @@
...
+
+```ts title="amplify/functions/say-hello/resource.ts"
+import { defineFunction } from "@aws-amplify/backend";
+import { DockerImage, Duration } from "aws-cdk-lib";
+import { Code, Function, Runtime } from "aws-cdk-lib/aws-lambda";
+import { execSync } from "child_process";
+import * as path from "path";
+import { fileURLToPath } from "url";
```
Comment (josefaidt) [inline src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx:55]:
```suggestion
import { execSync } from "node:child_process";
import * as path from "node:path";
import { fileURLToPath } from "node:url";
import { defineFunction } from "@aws-amplify/backend";
import { DockerImage, Duration } from "aws-cdk-lib";
import { Code, Function, Runtime } from "aws-cdk-lib/aws-lambda";
```

ordering nit for node imports

File: src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx [hunk 9]
```diff
@@ -0,0 +1,238 @@
...
+def handler(event, context):
+  return {
+      "statusCode": 200,
+      "body": json.dumps({
+          "message": "Hello World",
+      }),
+  }
+```
+
+The handler file _must_ export a function named "handler". This is the entry point to your function. For more information on writing functions, refer to the [AWS documentation for Lambda function handlers using python](https://docs.aws.amazon.com/lambda/latest/dg/python-handler.html).
```
Comment (josefaidt) [inline src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx:97]:
```suggestion
The handler file _must_ export a function named "handler". This is the entry point to your function. For more information on writing functions, refer to the [AWS documentation for Lambda function handlers using Python](https://docs.aws.amazon.com/lambda/latest/dg/python-handler.html).
```

casing nit for proper noun

File: src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx [hunk 10]
```diff
@@ -0,0 +1,238 @@
...
+
+If you need python packages, you can add them to a `requirements.txt` file in the same directory as your handler file. The `bundling` option in the `Code.fromAsset` method will install these packages for you.
```
Comment (josefaidt) [inline src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx:99]:
```suggestion
If you need Python packages, you can add them to a `requirements.txt` file in the same directory as your handler file. The `bundling` option in the `Code.fromAsset` method will install these packages for you.
```

File: src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx [hunk 11]
```diff
@@ -0,0 +1,238 @@
...
+```txt title="amplify/functions/say-hello/requirements.txt"
+request==2.25.1
+some-other-package>=1.0.0
+```
+
+You're now ready to deploy your python function. Next is the same process as the Node.js/TypeScript function. Go to [Common steps for all languages](#common-steps-for-all-languages) to continue.
+
+# Go functions
+To get started, install go package from npm. go to your backend directory, you should see amplify folder inside it. and run the following command:
+```npm i @aws-cdk/aws-lambda-go-alpha```
```
Comment (josefaidt) [inline src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx:111]: is this dependency still relevant?

File: src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx [hunk 12]
```diff
@@ -0,0 +1,238 @@
...
+
+ create a new directory and a resource file, `amplify/functions/say-hello/resource.ts`. Then, define the Function with `defineFunction`:
```
Comment (josefaidt) [inline src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx:113]:
```suggestion
Create a new directory and a resource file, `amplify/functions/say-hello/resource.ts`. Then, define the function with `defineFunction`:
```

File: src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx [hunk 13]
```diff
@@ -0,0 +1,238 @@
...
+Create a `requirements.txt` file in the same directory as your handler file. This file should contain the names of the packages you want to install. For example:
+
+```txt title="amplify/functions/say-hello/requirements.txt"
+request==2.25.1
+some-other-package>=1.0.0
+```
+
+You're now ready to deploy your python function. Next is the same process as the Node.js/TypeScript function. Go to [Common steps for all languages](#common-steps-for-all-languages) to continue.
+
+# Go functions
```
Comment (josefaidt) [inline src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx:109]:
```suggestion
## Go
```

File: src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx [hunk 14]
```diff
@@ -0,0 +1,238 @@
...
+
+ create a new directory and a resource file, `amplify/functions/say-hello/resource.ts`. Then, define the Function with `defineFunction`:
+
+```ts title="amplify/functions/say-hello/resource.ts"
+import { defineFunction } from "@aws-amplify/backend";
+import { DockerImage, Duration } from "aws-cdk-lib";
+import { Code, Function, Runtime } from "aws-cdk-lib/aws-lambda";
+import { execSync } from "child_process";
+import * as path from "path";
+import { fileURLToPath } from "url";
```
Comment (josefaidt) [inline src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx:121]:
```suggestion
import { execSync } from "node:child_process";
import * as path from "node:path";
import { fileURLToPath } from "node:url";
import { defineFunction } from "@aws-amplify/backend";
import { DockerImage, Duration } from "aws-cdk-lib";
import { Code, Function, Runtime } from "aws-cdk-lib/aws-lambda";
```

File: src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx [hunk 15]
```diff
@@ -0,0 +1,238 @@
...
+	return "Hello World!", nil
+}
+
+func main() {
+	lambda.Start(HandleRequest)
+}
+```
+
+Then you should run the following command to build the go function:
+```bash title="terminal"
```
Comment (josefaidt) [inline src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx:185]:
````suggestion
```bash title="Terminal" showLineNumbers={false}
````

File: src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx [hunk 16]
```diff
@@ -0,0 +1,238 @@
...
+go mod init lambda
+```
+then run to install the dependencies.
+
+```bash title="terminal"
```
Comment (josefaidt) [inline src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx:190]:
````suggestion
```bash title="Terminal" showLineNumbers={false}
````

File: src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx [hunk 17]
```diff
@@ -0,0 +1,238 @@
...
+go mod tidy
+```
+
+You're now ready to deploy your golang function. Next is the same process as the Node.js/TypeScript function.
+
+# Common steps for all languages
```
Comment (josefaidt) [inline src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx:196]:
```suggestion
## Common steps for all languages
```

File: src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx [hunk 18]
```diff
@@ -0,0 +1,238 @@
...
+Lastly, this function needs to be added to your backend.
```
Comment (josefaidt) [inline src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx:197]:
```suggestion
## Common steps for all languages

Regardless of the language used, your function needs to be added to your backend.
```

File: src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx [hunk 19]
```diff
@@ -0,0 +1,238 @@
...
+
+defineBackend({
+  // highlight-next-line
+  sayHelloFunctionHandler,
+});
+```
+
+Now when you run `npx ampx sandbox` or deploy your app on Amplify, it will include your Function.
+
+To invoke your Function, we recommend adding your [Function as a handler for a custom query with your Amplify Data resource](/[platform]/build-a-backend/data/custom-business-logic/). This will enable you to strongly type Function arguments and the return statement, and use this to author your Function's business logic. To get started, open your `amplify/data/resource.ts` file and specify a new query in your schema:
```
Comment (josefaidt) [inline src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx:212]: can we reword this to remove the language around strongly typed handlers? that will be a typescript-specific feature

File: src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx [hunk 20]
```diff
@@ -0,0 +1,238 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Custom functions',
+  description:
+    'Use another AWS Lambda runtimes like python, golang to perform tasks and customize workflows.',
```
Comment (josefaidt) [inline src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx:6]:
```suggestion
    'Use another AWS Lambda runtimes like Python, Golang to perform tasks and customize workflows.',
```

File: src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx (hunk 19 above)
Comment (MarlonJD) [reply src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx:212]: Actually, it's just works as handler, it's important use case for Python or Go. It would be nice to have an example here.

**Comment by MarlonJD** [comment]
Hi @josefaidt,
Thank you so much for your kind words and for taking the time to review this!
I really appreciate your feedback and have made adjustments based on your comments to refine the language. Please let me know if there’s anything else you'd like to tweak or improve further.

File: src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx [hunk 21]
```diff
@@ -0,0 +1,233 @@
...
+```bash title="terminal" showLineNumbers={false}
+go mod tidy
+```
+
+You're now ready to deploy your golang function. Next is the same process as the Node.js/TypeScript function.
+
+## Common steps for all languages  
+
+Regardless of the language used, your function needs to be added to your backend.
+// highlight-next-line
```
Comment (josefaidt) [inline src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx:196]:
````suggestion
```ts title="amplify/backend.ts"
// highlight-next-line
````

File: src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx [hunk 22]
```diff
@@ -0,0 +1,233 @@
...
+## Go
+To get started, Create a new directory and a resource file, `amplify/functions/say-hello/resource.ts`. Then, define the function with `defineFunction`:  
+
+```ts title="amplify/functions/say-hello/resource.ts"
+import { defineFunction } from "@aws-amplify/backend";
+import { DockerImage, Duration } from "aws-cdk-lib";
+import { Code, Function, Runtime } from "aws-cdk-lib/aws-lambda";
+import { execSync } from "node:child_process";
+import * as path from "node:path";
+import { fileURLToPath } from "node:url";
```
Comment (josefaidt) [inline src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx:118]:
```suggestion
import { execSync } from "node:child_process";
import * as path from "node:path";
import { fileURLToPath } from "node:url";
import { defineFunction } from "@aws-amplify/backend";
import { DockerImage, Duration } from "aws-cdk-lib";
import { Code, Function, Runtime } from "aws-cdk-lib/aws-lambda";
```

apologies, missed this one on reordering in my previous suggestions

File: src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx [hunk 23]
```diff
@@ -0,0 +1,233 @@
...
+  sayHelloFunctionHandler,
+});
+```
+
+Now when you run `npx ampx sandbox` or deploy your app on Amplify, it will include your Function.
+
+To invoke your Function, we recommend adding your [Function as a handler for a custom query with your Amplify Data resource](/[platform]/build-a-backend/data/custom-business-logic/). This will enable you to strongly type Function arguments and the return statement, and use this to author your Function's business logic. To get started, open your `amplify/data/resource.ts` file and specify a new query in your schema:
+
+```ts title="amplify/data/resource.ts"
+Now when you run `npx ampx sandbox` or deploy your app on Amplify, it will include your function.
```
Comment (josefaidt) [inline src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx:210]:
```suggestion
```

File: src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx [hunk 24]
```diff
@@ -0,0 +1,233 @@
...
+
+defineBackend({
+  // highlight-next-line
+  sayHelloFunctionHandler,
+});
+```
+
+Now when you run `npx ampx sandbox` or deploy your app on Amplify, it will include your Function.
+
+To invoke your Function, we recommend adding your [Function as a handler for a custom query with your Amplify Data resource](/[platform]/build-a-backend/data/custom-business-logic/). This will enable you to strongly type Function arguments and the return statement, and use this to author your Function's business logic. To get started, open your `amplify/data/resource.ts` file and specify a new query in your schema:
```
Comment (josefaidt) [inline src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx:207]:
```suggestion
To invoke your function, we recommend adding your [function as a handler for a custom query with your Amplify Data resource](/[platform]/build-a-backend/data/custom-business-logic/). To get started, open your `amplify/data/resource.ts` file and specify a new query in your schema:
```

**Review by josefaidt — DISMISSED** [review]

**Review by sobolk — DISMISSED** [review]

File: src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx [hunk 25]
```diff
@@ -0,0 +1,238 @@
...
+// highlight-next-line
+import { sayHelloFunctionHandler } from './functions/say-hello/resource';
+
+defineBackend({
+  // highlight-next-line
+  sayHelloFunctionHandler,
+});
+```
+
+Now when you run `npx ampx sandbox` or deploy your app on Amplify, it will include your Function.
```
[the remark this answers is not in the document]
Comment (jordanvn) [reply src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx:206]: Unresolving, as this seems like a change we'd want to make

[the remark this answers is not in the document]
Comment (MarlonJD) [reply src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx]: Fixed

**Review by sobolk — APPROVED** [review]

File: src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx (hunk 5 above)
Comment (jordanvn) [reply src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx:43]: Unresolving, as I think this still makes sense to remove. Is the mentioning of this relevant information on line 33 not sufficient?

**Review by josefaidt — APPROVED** [review]

File: src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx [hunk 26]
```diff
@@ -0,0 +1,230 @@
...
+import { Code, Function, Runtime } from "aws-cdk-lib/aws-lambda";
+import { defineFunction } from "@aws-amplify/backend";
+import { DockerImage, Duration } from "aws-cdk-lib";
+import { Code, Function, Runtime } from "aws-cdk-lib/aws-lambda";
+export const sayHelloFunctionHandler = defineFunction(
+  (scope) =>
+    new Function(scope, "say-hello", {
+      handler: "bootstrap",
+      runtime: Runtime.PROVIDED_AL2023, // or any other python version
+      timeout: Duration.seconds(3), //  default is 3 seconds
```
Comment (jordanvn) [inline src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx:124]: Are these comments still correct? They seem copied over from the previous Python example without change

File: src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx [hunk 27]
```diff
@@ -0,0 +1,230 @@
...
+import { DockerImage, Duration } from "aws-cdk-lib";
+import { Code, Function, Runtime } from "aws-cdk-lib/aws-lambda";
+import { defineFunction } from "@aws-amplify/backend";
+import { DockerImage, Duration } from "aws-cdk-lib";
+import { Code, Function, Runtime } from "aws-cdk-lib/aws-lambda";
+export const sayHelloFunctionHandler = defineFunction(
+  (scope) =>
+    new Function(scope, "say-hello", {
+      handler: "bootstrap",
+      runtime: Runtime.PROVIDED_AL2023, // or any other python version
```
Comment (josefaidt) [inline src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx:123]:
```suggestion
      runtime: Runtime.PROVIDED_AL2023,
```

File: src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx (hunk 26 above)
Comment (josefaidt) [reply src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx:124]: ope, good call, added a suggestion to remove

Comment (jordanvn) [reply src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx]: Thanks. Also on the fence about mentioning default timeout value again, especially since in this example the timeout is already being set to the default value

**Review by josefaidt — APPROVED** [review]

File: src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx (hunk 26 above)
Comment (josefaidt) [reply src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx]: I thought of removing it but left it for those that visit this page and jump straight to the Go example

Comment (jordanvn) [reply src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx]: I was thinking the same thing. Still potentially useful for that use case. Keeping it is fine with me

**Review by jordanvn — APPROVED** [review]

**Review by sobolk — APPROVED** [review]
